src/Raven.Studio/typescript/models/database/tasks/prefixPathModel.ts

```diff
@@ -0,0 +1,38 @@
+﻿import jsonUtil = require("common/jsonUtil");
+
+class prefixPathModel {
+    path = ko.observable<string>();
+    validationGroup: KnockoutValidationGroup;
+
+    constructor(prefixPath: string) {
+
+        this.path(prefixPath);
+        this.initValidation();
+    }
+
+    initValidation() {
+        this.path.extend({
+            validation: [
+                {
+                    validator: () => {
+                        if (this.path()) {
+                            const pathLength = this.path().length;
```

Comment:
please assert on length before getting last and previous character.

Reply:
done

Reply:
this is still invalid. You are checking if path length === 1 and then access last and previous character.